pkg/resources/cruisecontrol/configmap.go

```diff
@@ -157,6 +161,21 @@ func GenerateCapacityConfig(kafkaCluster *v1beta1.KafkaCluster, log logr.Logger,
 		}
 	}
 
+	capacityConfig := CapacityConfig{}
+	BrokerCapacities, err := appendGeneratedBrokerCapacities(kafkaCluster, log, nil)
+	if err != nil {
+		return "", err
+	}
+	capacityConfig.BrokerCapacities = append(capacityConfig.BrokerCapacities, BrokerCapacities...)
+	result, err := json.MarshalIndent(capacityConfig, "", "    ")
+	if err != nil {
+		log.Error(err, "Could not marshal cruise control capacity config")
```

Comment:
logging the error is redundant as the returned error will be logged.
```suggestion
		return "", err
```